<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoUIClass</class>
 <widget class="QWidget" name="VideoUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>970</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#VideoUIClass
{
	background-color: rgb(255, 255, 255);
}
QLabel{
	font: 14pt &quot;宋体&quot;;
	color: rgb(0, 0, 0);

}

#playBtn
{
	image: url(:/resources/Resources/play.png);
}
#playBtn:hover
{
	image: url(:/resources/Resources/play.png);
	color: rgb(230, 255, 222);
}
#playBtn:pressed
{
	image: url(:/resources/Resources/play.png);
	color: rgb(180, 255, 152);
}
#pauseBtn:hover
{
	image: url(:/resources/Resources/pause.png);
	color: rgb(230, 255, 222);
}
#pauseBtn:pressed
{
	image: url(:/resources/Resources/pause.png);
	color: rgb(180, 255, 152);
}
#pauseBtn
{
	image: url(:/resources/Resources/pause.png);
}
</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>20</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;等线&quot;;
color: rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="VideoWidget" name="des">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>20</y>
     <width>580</width>
     <height>420</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="exportBtn">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>630</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>导出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>510</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pauseBtn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>510</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="setBtn">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>670</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>620</y>
     <width>471</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="verticalSpacing">
     <number>6</number>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>对比度</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>图形金字塔</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QLineEdit" name="pyDown"/>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="pyUp"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="styleSheet">
         <string notr="true">font: 6pt &quot;Agency FB&quot;;</string>
        </property>
        <property name="text">
         <string>高斯金字塔次数</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_10">
        <property name="styleSheet">
         <string notr="true">font: 6pt &quot;Agency FB&quot;;</string>
        </property>
        <property name="text">
         <string>拉普拉斯金字塔次数</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>镜像</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>亮度</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="contrast"/>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="rotateSelect">
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>90</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>180</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>270</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="1">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="6">
       <widget class="QLabel" name="label_13">
        <property name="styleSheet">
         <string notr="true">font: 6pt &quot;Agency FB&quot;;</string>
        </property>
        <property name="text">
         <string>高度</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QDoubleSpinBox" name="clipW">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>2560.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_12">
        <property name="styleSheet">
         <string notr="true">font: 6pt &quot;Agency FB&quot;;</string>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_15">
        <property name="styleSheet">
         <string notr="true">font: 6pt &quot;Agency FB&quot;;</string>
        </property>
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QDoubleSpinBox" name="clipH">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>2560.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="clipX">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>2560.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_14">
        <property name="styleSheet">
         <string notr="true">font: 6pt &quot;Agency FB&quot;;</string>
        </property>
        <property name="text">
         <string>宽度</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="clipY">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>2560.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="bright">
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>裁剪</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>旋转</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="flipSelect">
      <item>
       <property name="text">
        <string>无</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>左右镜像</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>上下镜像</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>上下左右镜像</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="previewBtn">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>710</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>预览</string>
   </property>
  </widget>
  <widget class="QSlider" name="splitEndSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>561</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 127);
border-radius:10px;</string>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="value">
    <number>999</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="splitStartSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>561</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 127);
border-radius:10px;</string>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="VideoWidget" name="src1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>580</width>
     <height>420</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="playSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>561</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="srcList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>151</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>600</y>
     <width>341</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="2" column="0">
     <widget class="QPushButton" name="markBtn">
      <property name="text">
       <string>水印</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>宽度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="setWidth"/>
      </item>
      <item>
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>高度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="setHight"/>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="4">
       <widget class="QLabel" name="label_19">
        <property name="styleSheet">
         <string notr="true">font: 6pt &quot;Agency FB&quot;;</string>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="mx">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>2560.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_17">
        <property name="styleSheet">
         <string notr="true">font: 6pt &quot;Agency FB&quot;;</string>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="my">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>2560.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QDoubleSpinBox" name="mApha">
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_18">
        <property name="styleSheet">
         <string notr="true">font: 6pt &quot;Agency FB&quot;;</string>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="graySelect">
      <item>
       <property name="text">
        <string>RGB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>灰度图</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>图像类型</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="open2">
      <property name="text">
       <string>打开第二路视频</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>尺寸</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QComboBox" name="src2Select">
        <item>
         <property name="text">
          <string>融合</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>拼接</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="src2Apha"/>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="concatBtn">
      <property name="text">
       <string>拼接</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">videowidget.h</header>
   <slots>
    <slot>concat()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="VideoUI.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>VideoUIClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1264</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>1235</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playBtn</sender>
   <signal>clicked()</signal>
   <receiver>VideoUIClass</receiver>
   <slot>play()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pauseBtn</sender>
   <signal>clicked()</signal>
   <receiver>VideoUIClass</receiver>
   <slot>pause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>489</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>markBtn</sender>
   <signal>clicked()</signal>
   <receiver>VideoUIClass</receiver>
   <slot>setMark()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>686</x>
     <y>793</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open2</sender>
   <signal>clicked()</signal>
   <receiver>VideoUIClass</receiver>
   <slot>open2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>709</x>
     <y>822</y>
    </hint>
    <hint type="destinationlabel">
     <x>566</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>concatBtn</sender>
   <signal>clicked()</signal>
   <receiver>VideoUIClass</receiver>
   <slot>concat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>851</y>
    </hint>
    <hint type="destinationlabel">
     <x>557</x>
     <y>-152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>splitStartSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>VideoUIClass</receiver>
   <slot>splitLeft(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>521</x>
     <y>-116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>VideoUIClass</receiver>
   <slot>sliderRelease()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playSlider</sender>
   <signal>sliderPressed()</signal>
   <receiver>VideoUIClass</receiver>
   <slot>sliderPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>449</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>splitEndSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>VideoUIClass</receiver>
   <slot>splitRight(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>475</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>-173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>VideoUIClass</receiver>
   <slot>setPos(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>srcList</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>VideoUIClass</receiver>
   <slot>setCurrentVideoRow(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>682</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>-286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setBtn</sender>
   <signal>clicked()</signal>
   <receiver>VideoUIClass</receiver>
   <slot>setFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1179</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>608</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exportBtn</sender>
   <signal>clicked()</signal>
   <receiver>VideoUIClass</receiver>
   <slot>exportVideo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1146</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>718</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>previewBtn</sender>
   <signal>clicked()</signal>
   <receiver>VideoUIClass</receiver>
   <slot>preview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1164</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>1272</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot1()</slot>
  <slot>open()</slot>
  <slot>sliderPress()</slot>
  <slot>sliderRelease()</slot>
  <slot>setPos(int)</slot>
  <slot>setFilter()</slot>
  <slot>exportVideo()</slot>
  <slot>play()</slot>
  <slot>pause()</slot>
  <slot>setMark()</slot>
  <slot>open2()</slot>
  <slot>concat()</slot>
  <slot>splitLeft(int)</slot>
  <slot>splitRight(int)</slot>
  <slot>preview()</slot>
  <slot>setCurrentVideoRow(int)</slot>
 </slots>
</ui>
